lib/adal-angular.js

```diff
@@ -89,6 +89,7 @@ if (typeof module !== 'undefined' && module.exports) {
                         }
 
                         if (requestInfo.requestType !== _adal.REQUEST_TYPE.LOGIN) {
+                            _adal._renewActive = false;
```

Comment:
@ischrei Thanks for catching this! I agree that we need to report the error somehow but the loginFailure event does not seem to be the right one to be broadcasted. This error is likely to occur at renew time rather than login. We are on a tight schedule to release 1.0.9, I will fix this up in next release. If this is not yet reported, please feel free to open an issue.